styles.css

```diff
@@ -111,8 +111,729 @@ nav i {
   margin-left: 15px;
 }
 
-figure {
+.fig1 {
   position: absolute;
   top: 470px;
   right: -40px;
 }
+
+.my-recent-works {
+  background: #f5f5f5;
+  display: flex;
+  flex-direction: column;
+  left: 500px;
+  top: 700px;
+  position: absolute;
+  height: 3455px;
+  width: 375px;
+}
+
+.recent-work h2 {
+  color: #1c1a19;
+  font-size: 40px;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0.37px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: "Crete Round", "Times New Roman", Times, serif;
+  position: absolute;
+  left: 24px;
+  top: 120px;
+}
+
+.recent-work figure {
+  position: absolute;
+  left: -16px;
+  top: 260px;
+}
+
+.recent-work h3 {
+  color: #091e42;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0.37px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: "Crete Round", "Times New Roman", Times, serif;
+  position: absolute;
+  left: 24px;
+  top: 530px;
+}
+
+.recent-work p {
+  color: #091e42;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.37px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  position: absolute;
+  left: 24px;
+  top: 578px;
+}
+
+.recent-work span :nth-child(n) {
+  position: absolute;
+  left: -60px;
+  top: 715px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  border: 1px solid #8993a4;
+  background: #f5f5f5;
+  color: #000;
+}
+
+.recent-work span :nth-child(2) {
+  left: -1px;
+}
+
+.recent-work span :nth-child(3) {
+  left: 65px;
+}
+
+.recent-work span :nth-child(4) {
+  left: 170px;
+}
+
+button {
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  border: 0;
+  padding: 0.7rem;
+  margin-left: 85px;
+  margin-top: 20px;
+}
+
+.about-me button {
+  position: absolute;
+  left: -60px;
+  top: 370px;
+  margin-right: 24px;
+  color: #fff;
+  text-decoration: none;
+}
+
+.recent-work a button {
+  position: absolute;
+  left: -60px;
+  top: 770px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 700;
+}
+
+.about-me p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  position: absolute;
+  left: 24px;
+  top: 220px;
+}
+
+.get-in-touch p {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-me a :hover {
+  color: #ff6b00;
+}
```

Comment:
- [ ] Please make sure that the button's styling: Active, Hover, and Pressed work as expected.

![image](https://user-images.githubusercontent.com/58642949/192124212-d5508aa0-1350-461b-bfb6-17a95ced814c.png)